docs/source/techniques/DirectILL.rst

```diff
@@ -705,4 +743,101 @@ Output:
     SofQW_1.5K: Q range: 0.0...9.21A; W range -97.0...7.62meV
     SofQW_50K: Q range: 0.0...9.19A; W range -96.6...7.62meV
 
+And the same reduction, but with the use of the :ref:`algm-DirectILLAutoProcess`:
+
+.. testsetup:: FullExampleAutoProcess
+
+    config.setFacility('ILL')
+    config.appendDataSearchSubDir('ILL/IN4/')
+
+.. testcode:: FullExampleAutoProcess
+
+    vanadium_runs = 'ILL/IN4/085801-085802'
```

Comment:
Just a nitpick, if the `ILL/IN4` is added as a subdirectory, can we omit them in these paths to make it more appealing to the eye? Also `.nxs` extensions can be skipped.